journal/modernized/07-chapter-6.adoc

```diff
@@ -90,6 +92,7 @@ wills and flesh and held the profession of truth in unrighteousness.
 I could not get from under this burden,
 until it pleased God to send his servant George Fox,
 to set up men`'s and women`'s meetings, and then I was eased; of which hereafter.
+//Is the "of which hereafter" above okay? It was marked in the book.
```

Comment:
yeah, it's not a mistake (it means "more on that later"), but it does seem awkward. i made a note to think about it and maybe come back later.  let's leave it as is for now.